app/components/avatar_component.rb

```diff
@@ -6,7 +6,8 @@ class AvatarComponent < ViewComponent::Base
   def initialize(avatarable:, variant: nil, classes: nil, data: {})
     @avatarable = avatarable
     @variant = variant
-    @classes = classes
+    @classes = (classes || "h-24 w-24 sm:h-32 sm:w-32 ring-4 ring-white")
+      .concat(" ", "object-cover rounded-full")
```

Comment:
Let's move all of the class logic into this file. Right now we have it split between the HTML part and the Ruby file.

I like the idea of exposing `classes` via a method instead of the `attr_reader`, too. For example,

In the initializer, `@classes = classes`. Then,

```ruby
def classes
  [
    (@classes || "h-24 w-24 sm:h-32 sm:w-32 ring-4 ring-white"),
    "object-cover rounded-full",
    "bg-navy": Feature.enabled?(:redesign),
    "bg-gray-300": !Feature.enabled?(:redesign)
  ]
end
```

Reply:
I'm not sure this would work with the current interface setup for `initialize`. I have to leave for the evening, but I can try to figure this out later.

Reply:
Sorry for missing out on this today. I woke up under the weather today, and I basically spent the whole day trying to shake that off.